src/app/models/result-summary/result-summary.ts

```diff
@@ -0,0 +1,68 @@
+export class ResultSummary {
+  id: number;
+
+  uri: string;
+
+  httpMethod: string;
+
+  response25Percentile: number;
+
+  response50Percentile: number;
+
+  responseAvg: number;
+
+  response75Percentile: number;
+
+  responseMax: number;
+
+  failCount: number;
+
+  successFailPercentage: number;
+
+  reqPerSec: number;
+
+  dataReference: string;
```

Comment:
I recommend reaching out to the backend group, as the current state of the application is not quite using the S3. However, they might transition to it today.

Reply:
The transition has not yet occured